src/main.tsx

```diff
@@ -2,9 +2,9 @@ import { StrictMode } from "react";
 import { createRoot } from "react-dom/client";
 import { BrowserRouter, Routes, Route } from "react-router";
 import "./index.css";
-import Layout from "./components/Layout.tsx";
-import Home from "./routes/Home.tsx";
-import About from "./routes/About.tsx";
+import Layout from "./components/Layout/index.tsx";
+import Home from "./routes/Home/index.tsx";
+import About from "./routes/About/index.tsx";
```

Comment:
Because these components are exported as default, I believe we can change all of these to leave off the 
`/index.tsx`.
E.g.
```js
import Home from "./routes/Home";
```